search-service/src/main/java/de/cxp/ocs/elasticsearch/facets/TermFacetCreator.java

```diff
@@ -30,7 +31,7 @@ public class TermFacetCreator extends NestedFacetCreator {
 	@Setter
 	private int									maxFacetValues	= 100;
 	private final Locale						locale;
-	private final Map<String, FacetEntrySorter>	facetSorters	= new HashMap<>();
+	private final Map<String, FacetEntrySorter>	facetSorters	= new ConcurrentHashMap<>();
```

Comment:
Why must this be now a ConcurrentHashMap?

Reply:
This is my commit. I changed the PR target to remove this commit from here.
But as an explanation: It fixes a ConcurrentModificationException we experienced at a customer. See [commit message](https://github.com/CommerceExperts/open-commerce-search/commit/4b65b35874fe9ce009980e2612fbd479ebb0d637).